pkgs/development/python-modules/grapheme/default.nix

```diff
@@ -0,0 +1,20 @@
+{ lib, buildPythonPackage, fetchPypi, pythonOlder }:
+
+buildPythonPackage rec {
+  pname = "grapheme";
+  version = "0.6.0";
+
+  disabled = pythonOlder "3.7";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1jiwc3w05c8kh22s3zk7a8km8na3plqc5zimb2qcyxxy3grbkhj4";
+  };
```

Comment:
Pypi sdist is missing tests, please run the tests from github.

There are no tests included. Please try to checkout from source and check if they have unit tests, and try to run them. Unit tests give a good indication that they package has a high degree of validity and correctness given the python package set.

If tests are not available, then please use `pythonImportsCheck` to import the most important modules. This isn't as good as unit tests, but will usually give a good indication of run-time errors. Please see [pythonImportsCheck documentation](https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#using-pythonimportscheck) for more information.

```

----------------------------------------------------------------------
Ran 0 tests in 0.000s
```

Reply:
added ``pythonImportsCheck = [ "grapheme" ];`` as v0.6.0 is only released on pypi.

Reply:
Perhaps we get tags.

https://github.com/alvinlindstam/grapheme/issues/18

Reply:
No response from upstream.